types/index.d.ts

```diff
@@ -179,19 +171,15 @@ export interface RenderHookResult<Result, Props> {
   unmount: () => void
 }
 
-export interface BaseRenderHookOptions<
+/** @deprecated */
+export type BaseRenderHookOptions<
```

Comment:
Just so I'll understand, what's the motivation with changing this one from an `interface` to a `type`?

Reply:
it's just an alias for `RenderHookOptions` whereas an `interface` would create a new type.